2017/Cambridge-OMERO-API/index.html

```diff
@@ -0,0 +1,268 @@
+                        8th December, 2017 <br>
+                        Will Moore, Balaji Ramalingam, Petr Walczysko
+                    </p>
+                </section>
+                
+                <section>
+                    <h2>Outline of the Day</h2>
+                    <ol>
+                        <li>Presentation
+                            <ul>
+                                <li>What is OMERO?</li>
+                                <li>OMERO model</li>
+                                <li>OMERO API
+                            </ul>
+                        </li>
+                        <li>ImageJ/Fiji as an OMERO client</li>
+                        <li>OMERO.scripts with SciPy
+                            <ul><li>OMERO.figure scripting with JSON</li></ul>
+                        </li>
+                        <li>Jupyter Notebooks
+                            <ul><li>Python/CellProfiler | R</li></ul>
+                        </li>
+                    </ol>
+                </section>
+
+
+                <section>
+                    <h2>OMERO is a server with clients</h2>
+
+                    <img src="https://downloads.openmicroscopy.org/presentations/2014/technique_tapas_nov_5/images/omero-platform-simple.png">
+                </section>
+
+
+                <section data-background="https://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/webclient3.png">
+                </section>
+
+
+                <section>
+                    <h2>OMERO model objects</h2>
+                    <img src="images/developer-model-pdi.png">
+                    <br>
+                    See <a href="http://docs.openmicroscopy.org/omero/5.4.0/developers/Model/EveryObject.html">
+                        All Objects
+                    </a>
+                </section>
+
+
+
+                <section>
+                    <h2>Remote Objects & Proxy Services</h2>
+                    <img src="images/omero-proxy-objects.png"><br>
+                    <a href="http://docs.openmicroscopy.org/latest/omero/developers/index.html">
+                        Developer Documentation
+                    </a>
+                </section>
+
+
+                <section>
+                    <h2>Connecting to OMERO: Python setup</h2>
+                    <ul>
+                        <li>Download
+                            <a href="https://downloads.openmicroscopy.org/latest/omero5/py.zip">
+                                OMERO.py libs
+                            </a>, add to PYTHONPATH
+                        </li>
+                        <li>
+                            <code class="python">$ pip install zeroc-ice&gt;3.5,&lt;3.7</code>
+                        </li>
+                    </ul>
+                </section>
+
+
+                <section>
+                    <h2>Connecting to OMERO: Python code</h2>
+                    <pre><code class="python">import omero
```

Comment:
will you go over the "keepAlive" verbally 
I don't think you mentioned it on Friday